pkg/security/oauth2/plgd.go

```diff
@@ -12,14 +12,21 @@ import (
 
 // NewPlgdProvider creates OAuth client
 func NewPlgdProvider(ctx context.Context, config Config, logger log.Logger) (*PlgdProvider, error) {
-	config.ResponseMode = "query"
-	config.AccessType = "offline"
-	config.ResponseType = "code"
+	if config.ResponseMode == "" {
```

Comment:
@Trishia why you need to make this values configurable? None of these should be change from it's default values, so why to make it configurable?

Reply:
@ondrejtomcik  oauth struct has these values so I add default check which is safer source code for flexibility if developers want to change these configs.

Reply:
But he shouldn't, that's the point. Response type should be always code, and it shouldn't be updated. so the question is if you have a use-case when you need to change these values

Reply:
If you think it should not be changed, rollback to original codes you want.